controller/space_template.go

```diff
@@ -0,0 +1,26 @@
+package controller
+
+import (
+	"github.com/fabric8-services/fabric8-wit/app"
+	"github.com/goadesign/goa"
+)
+
+// SpaceTemplateController implements the space_template resource.
+type SpaceTemplateController struct {
+	*goa.Controller
+}
+
+// NewSpaceTemplateController creates a space_template controller.
+func NewSpaceTemplateController(service *goa.Service) *SpaceTemplateController {
+	return &SpaceTemplateController{Controller: service.NewController("SpaceTemplateController")}
+}
+
+// Show runs the show action.
+func (c *SpaceTemplateController) Show(ctx *app.ShowSpaceTemplateContext) error {
+	// SpaceTemplateController_Show: start_implement
+
+	// Put your logic here
```

Comment:
I am wondering if instead of having an useless endpoint that returns nil. We could have a temporal redirection to `/api/spacetemplates/template_ID/workitemtypegroups` based on the ID passed when ` /api/spacetemplates/template_ID`